src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -843,15 +856,12 @@ private LfNetwork create(int numCC, int numSC, Network network, List<Bus> buses,
             LOGGER.warn("Network {}: {} branches have been discarded because connected to same bus at both ends",
                     lfNetwork, report.branchesDiscardedBecauseConnectedToSameBusAtBothEnds);
         }
-        if (report.linesWithDifferentNominalVoltageAtBothEnds > 0) {
```

Comment:
why it is removed?

Reply:
this is dead code ... See https://github.com/powsybl/powsybl-open-loadflow/pull/1030#issuecomment-2112800316

Reply:
@jeandemanged suggested in a comment above to remove this because it is not used anymore

Reply:
I think it is a mistake @geofjamg what's happen here?

Reply:
From here https://github.com/powsybl/powsybl-open-loadflow/pull/658/files#diff-ae21874f7b466e7204256111d8e9f6b3f268bb6d2210efdf6b6c4d30ecf08f92